addons/sale/models/sale.py

```diff
@@ -1836,6 +1835,13 @@ def _get_protected_fields(self):
     def _onchange_product_id_set_customer_lead(self):
         pass
 
+    @api.onchange('price_unit')
```

Comment:
why you don't add in the declaration of the fields `digits='Product Price'` ?

Reply:
@fmdl [that's already defined](https://github.com/odoo/odoo/blob/14.0/addons/sale/models/sale.py#L1369) but I supposed that get effect only while saving information in database not in onchange case. Probably something to improve at ORM level them ?

Reply:
If this is a real problem, it should indeed be fixed at orm level and not in sale, unless you can prove it's only happening in sale (but I don't really see what could be the cause of such a strange behavior in the sale code :thinking:).

Reply:
@fmdl @Feyensv I've change the fix at ORM level.

Is that better ? should I remove the test in sale module or could we keep both (at orm level to ensure things are properly working in the orm, at business level to ensure this use case stay in place) ?